src/installer/tests/HostActivation.Tests/StartupHooks.cs

```diff
@@ -173,11 +173,14 @@ public void Muxer_activation_of_Empty_StartupHook_Variable_Succeeds()
             var startupHookVar = "";
             dotnet.Exec(appDll)
                 .EnvironmentVariable(startupHookVarName, startupHookVar)
+                .EnvironmentVariable(Constants.HostTracing.TraceLevelEnvironmentVariable, "1")
+                .EnvironmentVariable(Constants.HostTracing.VerbosityEnvironmentVariable, "2")
                 .CaptureStdOut()
                 .CaptureStdErr()
```

Comment:
This can be replaced with `.EnableTracingAndCaptureOutputs`, it does not set verbosity (so it defaults to everything), but that should work for this test as well.

Similarly in the other places.